### DISCUSSION TITLE
在线播放的底部按钮怎么使用啊

### DISCUSSION THREAD

**Discussion by Thereisnoname233** [AlistGo/alist#3211] [Q&A]
在线播放的底部按钮怎么用啊，点了没有反应...
<img width="269" alt="屏幕截图 2023-02-01 104610" src="https://user-images.githubusercontent.com/99064124/215933105-cc5cb500-c323-4cf8-9627-22a5a68319bc.png">

**Comment by anwen-anyi** [comment]
1. 你没安装相对应的播放器
 2. 你安装的绿色版没写进注册表调用不起来

**Reply by Thereisnoname233** [reply]
我重装了vlc，在vlc官网上下的windows 64bit版，但现在还是没反应...

**Reply by anwen-anyi** [reply]
> 我重装了vlc，在vlc官网上下的windows 64bit版，但现在还是没反应...

vlc不太清楚，我是用的potplayer,没有问题
![OK](http://gchat.qpic.cn/gchatpic_new/0/0-0-F3F4EDBEC5C4A6B08C74B56CE260807A/0)

**Reply by mengguyi** [reply]
我在win10下用vlc官方的msi装的也不能用
在Ubuntu下用apt装的就可以
所以还是win下注册表的问题，可能和vlc自己有关

**Comment by royccg** [comment]
我为什么连底部的这个调用 本地播放器的图标，都没有啊，这个要在设置里面选择什么吗？

**Reply by anwen-anyi** [reply]
如果你挂载的是 阿里云盘开放平台并且使用的是`Aliyun Video Previewer` 播放器 那就没用下面的那一排播放器
其他的云盘会有的

**Reply by royccg** [reply]
我使用是 阿里云盘的Open平台，但是 选用的`使用的是Aliyun Video Previewer 播放器`，这是在哪里选择的呢？我看了挂载的设置，也没有这个选项啊？谢谢，解答

**Reply by royccg** [reply]
> 我使用是 阿里云盘的Open平台，但是 选用的`使用的是Aliyun Video Previewer 播放器`，这是在哪里选择的呢？我看了挂载的设置，也没有这个选项啊？谢谢，解答

我找到了，谢谢解答，哈哈哈哈
